app/src/main/res/layout/activity_marketing.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this enclosing `ConstraintLayout` necessary?

Reply:
Nope